src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  gap: $gap;
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  &__day {
+    @include square($full-day-size);
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #eee;
+    border: 1px black solid;
+    transition-duration: 0.5s;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: pink;
+    }
+  }
+
+  @for $date from 1 through 31 {
+    &__day:nth-child(#{$date})::before {
+      content: "#{$date}";
+    }
+  }
+
+  @for $month-day-count from 28 through 31 {
+    &--month-length--#{$month-day-count} {
+      &__day:nth-child(n + #{$month-day-count + 1}) {
```

Comment:
This approach doesn't work because you have nested selectors. So you have the next result:
<img width="365" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/25587299/d6f906bf-05c1-4e19-8d62-524d46aab984">
You can use here just `:nth-child` without the element name:
```
 &--month-length--#{$month-day-count} {
      :nth-child(n + #{$month-day-count + 1}) {
```

Now we can't change the amount of the month days:
<img width="1321" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/25587299/b723b615-5fbb-42a0-bdf4-3fecdecb33eb">